server/index.js

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import cors from 'cors';
+import sqlite3 from 'sqlite3'; // Import sqlite3
+import blogControllerRoutes from './Controllers/blogControllers.js';
```

Comment:
where is `blogControllerRoutes` defined?

Reply:
This is where `blogControllerRoutes.js` is defined `import blogControllerRoutes from './Controllers/blogControllers.js';`
Actually, in `node.js` the index.js file role is as a `parent file`, Like if you want to run your server you have to execute only index.js file nothing else. Therefore, all the `routes` and `controller` and `CRUD operations` are being operated and called as an `instance` in index.js file.